### DISCUSSION TITLE
Question: Custom accent color with winui 2.6

### DISCUSSION THREAD

**Discussion by lhak** [microsoft/microsoft-ui-xaml#7460] [Q&A]
With previous winui versions it was sufficient to override SystemAccentColor and its variants (SystemAccentColorDark1...) to set a custom accent color for all controls. However, with winui 2.6 this only works partially. It seems that it is at least necessary to override AccentFillColorDefaultBrush, even though according to the xaml file (Common_themeresources_any.xaml) it should derive from SystemAccentColorLight2...

Can you please clarify (and update the documentation) which resources have to be set to have all controls show the custom accent color?

**Comment by StephenLPeters** [comment]
@chigy @alejo4000 @marksfoster can we get a list of all the new accent color resources.

**Comment by mdtauk** [comment]
Is there a way to provide a custom single accent colour, and for WinUI to generate the correct light and dark shades with contrast ratio protections?

If not generated by WinUI, then perhaps an updated Fluent Theme Editor?

**Comment by soumyamahunt** [comment]
> Is there a way to provide a custom single accent colour, and for WinUI to generate the correct light and dark shades with contrast ratio protections?
> 
> If not generated by WinUI, then perhaps an updated Fluent Theme Editor?

It used to be possible with [`ColorPaletteResources`](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.colorpaletteresources?view=winrt-20348) but it doesn't work anymore with WinUI 2.x. This was used in "Fluent Theme Editor" too. May be a new `ColorPaletteResources` API in WinUI namespace that can handle accent color customization will be required.

This issue blocks any app that provides accent color customization from adopting new designs for Windows 11.

**Comment by tipa** [comment]
I second this request.
I currently do have some code to manually create shades of the custom accent color a user is able to pick. However, these shades do not line up with the ones that Windows calculates. It would be highly appreciated if
- Windows/WinUI can calculate the shades automatically when supplying a custom accent color or
- the algorithm to calculate the correct shades is made available

**Comment by MajorXAML** [comment]
I would like to bump this topic since there is still no solution :/ 

@tipa 
Maybe you have succeeded?
